src/richie-front/scss/mixins/_generic.scss

```diff
@@ -0,0 +1,14 @@
+///
+/// Default base document font size
+/// Value stand on Bootstrap assertion of 16px default value from browsers
+/// But this is not a threat since rem will finally adapt relatively to any
+/// default value. However for sanity we use a variable.
+$global-base-font-size: 16px !default;
```

Comment:
Why not just use `$font-size-base` from bootstrap?

Reply:
For this function helper, you need a pixel value to computate the rem one and `$font-size-base` setting from bootstrap use `1rem`.